app/src/main/java/com/example/sun/utils/Constant.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.sun.utils
+
+object Constant {
+    const val BASE_URL = "http://api.weatherapi.com/v1/"
+    const val BASE_SEARCH_CURRENT_WEATHER = "current.json?"
+    const val BASE_SEARCH_FORECAST = "forecast.json?"
+    const val BASE_API_KEY = "key=" + "4abd7b21910340fd889235405242307"
```

Comment:
Không cộng string ở hằng số, để việc cộng string khi build URL
```suggestion
    const val BASE_API_KEY = "4abd7b21910340fd889235405242307"
```